pyproject.toml

```diff
@@ -1,21 +1,32 @@
-[tool.black]
+[tool.ruff]
 line-length = 120
-target-version = ["py37", "py38", "py39", "py310"]
```

Comment:
Should we keep `target-version` as 39 which is the lambda runtime version we use?